src/pages/admin/quizzes/QuizDetail/index.module.scss

```diff
@@ -1,14 +1,18 @@
 @use '../../../../styles/variables' as *;
 
+.cardContainer {
+  position: absolute;
+  height: 100%;
+  width: calc(100% - 300px);
+  left: 300px;
+  padding: 60px 40px 60px;
+}
+
```

Comment:
i can see this is being used in multiple files. can you put this in a global file that can be accessed so that we will just call the file